zio/src/main/scala/org/scanamo/ScanamoZio.scala

```diff
@@ -29,11 +29,11 @@ class ScanamoZio private (client: DynamoDbAsyncClient)
 
 object ScanamoZio {
   type DIO[+A] = IO[DynamoDbException, A]
+  type DStream[+A] = Stream[DynamoDbException, A]
 
   def apply(client: DynamoDbAsyncClient): ScanamoZio = new ScanamoZio(client)
 
-  val ToStream: IO[DynamoDbException, *] ~> Stream[DynamoDbException, *] =
-    new (IO[DynamoDbException, *] ~> Stream[DynamoDbException, *]) {
-      def apply[A](fa: IO[DynamoDbException, A]): Stream[DynamoDbException, A] = ZStream.fromEffect(fa)
-    }
+  val ToStream: DIO ~> DStream = new (DIO ~> DStream) {
+    def apply[A](fa: DIO[A]): DStream[A] = ZStream.fromEffect(fa)
+  }
```

Comment:
We've recently seen this problem before:

* https://github.com/scanamo/scanamo/pull/1805

As [mentioned there](https://github.com/scanamo/scanamo/pull/1805#discussion_r1731496767), this fixes a [compilation error](https://github.com/scanamo/scanamo/actions/runs/10582728962/job/29323114814?pr=1806#step:7:1027) relating to [Scala 3's syntax for wildcard arguments in types](https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html):

```
[error] -- [E006] Not Found Error: /home/runner/work/scanamo/scanamo/zio/src/main/scala/org/scanamo/ScanamoZio.scala:35:38 
[error] 35 |  val ToStream: IO[DynamoDbException, *] ~> Stream[DynamoDbException, *] =
[error]    |                                      ^
[error]    |                                      Not found: type *
```